grammars/less.cson

```diff
@@ -273,12 +273,12 @@
     "name": "keyword.other.important.css"
   }
   {
-    'match': '\\$|%|&|\\*|\\/|\\-\\-|\\-|\\+\\+|\\+|~|===|==|=|!=|!==|<=|>=|<<=|>>=|>>>=|<>|<|>|!|&&|\\|\\||\\?\\:|\\*=|(?<!\\()/=|%=|\\+=|\\-=|&=|\\^=|\\/\\b'
+    'match': '\\$|&|\\*|\\/|\\-|\\+|~|=|<=|>=|<|>|&&|\\?\\:|(?<!\\()/=|\\/\\b'
```

Comment:
> What is the `$` doing? I don't recall that having any specific meaning in less

`$` is not less operator. I just missed it.
I will _remove_ it in next commit.

---
> Is `&&` supported and not `||`?

`&&` is valid less syntax for [multiple `&`](http://lesscss.org/features/#parent-selectors-feature-multiple-).

Since there has been other regex for [preant-selector `&`](https://github.com/liuderchi/language-less/blob/3831bb5f7db0d305fa1f2fd3398392505cbbba19/grammars/less.cson#L37), IMHO _both_ `&` and `&&` could be _removed_ from this regex (`keyword.operator.less`).

**Question**: How do you think about it?

---
> ?: seems to be completely wrong.

`?:` Not found in official website.
I will _remove_ it next commit.

---
> `/=` shouldn't be supported

I missed it and will _remove_ it next commit.

---
> \\/\\b won't ever get matched because of the \\/ earlier on.

I will _remove_ `\\/\\b` in next commit.